contracts/0.8.25/vaults/StakingVault.sol

```diff
@@ -284,27 +298,27 @@ contract StakingVault is IStakingVault, IBeaconProxy, VaultBeaconChainDepositor,
         if (msg.sender != address(VAULT_HUB)) revert NotAuthorized("lock", msg.sender);
 
         VaultStorage storage $ = _getVaultStorage();
-        if ($.locked > _locked) revert LockedCannotBeDecreased(_locked);
+        if ($.locked > _locked) revert LockedCannotDecreaseOutsideOfReport($.locked, _locked);
 
-        $.locked = SafeCast.toUint128(_locked);
```

Comment:
Maybe, it's not worth to trust oracle here so much?

Reply:
I think we will definitely need a sanity checker here but for now we can leave the safecast.